app/domain/authentication/authn_jwt/consts.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+module Authentication
+  module AuthnJwt
+    PROVIDER_URI_RESOURCE_NAME = "provider-uri"
+    JWKS_URI_RESOURCE_NAME = "jwks-uri"
+    PROVIDER_URI_INTERFACE_NAME = PROVIDER_URI_RESOURCE_NAME.freeze
+    JWKS_URI_INTERFACE_NAME = JWKS_URI_RESOURCE_NAME.freeze
+    ISSUER_RESOURCE_NAME = "issuer"
+    IDENTITY_FIELD_VARIABLE = "token-app-property"
```

Comment:
why is this `IDENTITY_FIELD_VARIABLE` and not `TOKEN_APP_PROPERTY_VARIABLE`?

Reply:
renamed to TOKEN_APP_PROPERTY_VARIABLE